### PR TITLE
add testnets: Holeski, Arb Sepolia, OP Sepolia

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-####
Figma link (if any):

## Screen recordings / screenshots

adding testnets: Holeski, Arb Sepolia, OP Sepolia

### CODE REVIEW

**Review by greg-schrammel — APPROVED** [review]
👍

File: src/core/types/chains.ts [hunk 1]
```diff
@@ -36,19 +39,70 @@ export const hardhatOptimism: Chain = {
   testnet: true,
 };
 
+export const chainHoleski: Chain = {
```
Comment (DanielSinclair) [inline src/core/types/chains.ts:42]: for normalization we might want to use `Holesky` with `y` here

File: src/core/types/chains.ts [hunk 2]
```diff
@@ -36,19 +39,70 @@ export const hardhatOptimism: Chain = {
...
+  id: HOLESKI_CHAIN_ID,
+  name: 'Holesky',
+  network: 'holesky',
+  nativeCurrency: {
+    decimals: 18,
+    name: 'Holeski Ether',
```
Comment (DanielSinclair) [inline src/core/types/chains.ts:48]: Double checked the chain list and am seeing this as `Holesky Ether` elsewhere

File: src/core/types/chains.ts [hunk 3]
```diff
@@ -85,13 +165,16 @@ export const ChainNameDisplay = {
...
-  [ChainId['zora-testnet']]: chain.zoraTestnet.name,
+  [ChainId.holesky]: chainHoleski.name,
+  [ChainId.optimismGoerli]: chain.optimismGoerli.name,
+  [ChainId.optimismSepolia]: chainOptimismSepolia.name,
+  [ChainId.bscTestnet]: 'BSC Testnet',
+  [ChainId.polygonMumbai]: chain.polygonMumbai.name,
+  [ChainId.arbitrumGoerli]: chain.arbitrumGoerli.name,
+  [ChainId.arbitrumSepolia]: chainArbitrumSepolia.name,
+  [ChainId.baseGoerli]: chain.baseGoerli.name,
+  [ChainId.zoraTestnet]: chain.zoraTestnet.name,
```
Comment (DanielSinclair) [inline src/core/types/chains.ts:179]: Noticed `Zora Goerli` had testnet appended here; not exactly sure where that originates. 

Screenshot 2023-11-22 at 1 02 41 AM

**Review by DanielSinclair — APPROVED** [review]
Functionality looks good! 👌 
I pinged Wayne to see if we can get Base Sepolia and Zora Sepolia added at the same time. QuickNode supports Base Sepolia now, so I think we could add it here once our node gets deployed; up to you on order of events. I can sort out the env additions if you want to leave a placeholder

File: src/core/types/chains.ts (hunk 1 above)
Comment (estebanmino) [reply src/core/types/chains.ts:42]: oh lol will change

Comment (estebanmino) [reply src/core/types/chains.ts:42]: about the new chains, we can add them wen ready

File: src/core/types/chains.ts (hunk 3 above)
Comment (estebanmino) [reply src/core/types/chains.ts:179]: image

Comment (DanielSinclair) [reply src/core/types/chains.ts]: Nice, thank you!

**Pull request merged**